app/Http/Controllers/DashboardController.php

```diff
@@ -67,185 +71,77 @@ public function dev_layout()
      * @param  Request  $request
      * @return \Illuminate\Http\JsonResponse
      */
-    public function ajaxRefreshAll(Request $request)
+    public function refreshPage(Request $request)
     {
-        Validator::make($request->all(), [
-            'site_id' => 'required|int|max:255',
-            'location_id' => 'required|int|max:255',
-            'device_id' => 'required|int|max:255',
-        ])->validate();
-        
+        $request->validate([
+            'site_id' => 'required|integer|digits_between:1,7',
+            'location_id' => 'sometimes|required|integer|digits_between:1,7',
+            'device_id' => 'sometimes|required|integer|digits_between:1,7',
+            'offset' => 'sometimes|required|integer|digits_between:1,7',
+        ]);
+    
```

Comment:
Looks to be arbitrarily limiting index sizes to 7 digits. In the database the maximum index (id) is 10 digits.

Reply:
It was arbitrary, I didn't notice we had a max defined somewhere. Updating.